ydb/core/util/backoff.h

```diff
@@ -16,7 +17,35 @@ class TBackoffTimer {
 public:
     TBackoffTimer(ui64 initialMs, ui64 maxMs);
     ui64 NextBackoffMs();
+    TDuration Next();
     void Reset();
 };
 
+// exponential backoff with jitter and the iteration counter
+struct TBackoff {
+    static constexpr TDuration DefaultInitialDelay = TDuration::Seconds(1);
+    static constexpr TDuration DefaultMaxDelay = TDuration::Minutes(15);
+
+    // backoff with unlimited retries
+    TBackoff(TDuration initialDelay = DefaultInitialDelay, TDuration maxDelay = DefaultMaxDelay);
+    // backoff with limited retries
+    TBackoff(size_t maxRetries, TDuration initialDelay = DefaultInitialDelay, TDuration maxDelay = DefaultMaxDelay);
```

Comment:
```suggestion
    explicit TBackoff(TDuration initialDelay = DefaultInitialDelay, TDuration maxDelay = DefaultMaxDelay);
    // backoff with limited retries
    explicit TBackoff(size_t maxRetries, TDuration initialDelay = DefaultInitialDelay, TDuration maxDelay = DefaultMaxDelay);
```

может сделать explicit? Чтобы `5` неявно в `TBackoff` нельзя было преобразовать.